Tests/AppTests/Util+ext.swift

```diff
@@ -73,12 +73,14 @@ extension Result {
 
 extension String {
     var gh: String { "https://github.com/foo/\(self)" }
+    var sw: SwiftVersion { SwiftVersion(self)! }
 }
 
 
 extension Array where Element == String {
     var urls: [URL] { compactMap(URL.init(string:)) }
     var gh: Self { map(\.gh) }
+    var sw: [SwiftVersion] { map(\.sw) }
```

Comment:
It's a small point, but these are super cryptic. I know you value brevity, and I value verbosity, but I think `sw` goes  a little too far :)

Reply:
Ah, that should be `sv` really 🤣

But yes, that’s a quick fix overstaying its welcome. Let me change these to `asSwiftVersion` and `asGithubUrl` after lunch!